package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-flag-pack",
   "description": "A React flag pack",
-  "version": "0.0.15",
```

Comment:
Version will be automatically increased when publishing.

Reply:
The package automatically does this when npm install is ran. So we might wanna fix that then if that's unintended behavior.

Reply:
That's weird..? It shouldn't. Only when you run `npm update`

Reply:
Don't forget to change the readme too if you change properties. If the readme is redundant (because of the docs page) remove it completely. But having two separate sources of 'truth' is misleading.

_.e.g. I found a bug which is not a bug just a misspelling._